src/main/java/org/jenkinsci/plugins/gitlab/GitlabBuilds.java

```diff
@@ -49,7 +54,12 @@ public String build(GitlabCause cause, Map<String, String> customParameters, Git
     	if (hasCommitStatus(project, cause.getLastCommitId(), api)) {
         	shouldRun = false;
         }
-    	
+
+        if (lastNote != null && lastNote.getBody().equals(triggerComment)) {
+            LOGGER.info("Trigger comment found");
+            shouldRun = true;
```

Comment:
No, works like a charm )
It triggers only once. I am using it almost 6 days already for our project builds.